doc/tutorial.md

```diff
@@ -0,0 +1,213 @@
+# Tutorial
+
+## Introduction
+This tutorial shows the steps for creating and instantiating a Dlite entity and populating it with data. For this, we will use an example `csv` file containing data corresponding to time measurements from a solar panel. The example file has the following format:
+
+
+| time                 | Impp [A] | Isc [A] | MPP [W] | Vmpp [V] | Voc [V] |
+|----------------------|----------|---------|---------|----------|---------|
+| 2022-08-01 00:41:00  |  1.188   |   1.25  |  38.182 |   32.14  |  37.33  |
+|  ...                 |   ...    |   ...   |   ...   |    ...   |   ...   |
+
+
+
+## Writing a DLite Entity
+Let us write an Entity representing our solar panel measurement. We start by creating a `json` file called `myEntity.json`.
+
+### **Step 1**: Giving it a unique identifier
+Firstly, we must provide a unique identifier for our Entity. There are several ways of doing this:
+
+#### **1.  Uniform Resource Identifier (URI)**
+We can create and assign a unique Uniform Resource Identifier (URI). An URI has the form **namespace/version/nameOfEntity**, and it can be either written directly in the json file:
```

Comment:
```suggestion
We can create and assign a unique Uniform Resource Identifier (URI). A URI has the form **namespace/version/nameOfEntity**, and it can be either written directly in the json file:
```